app/models/folio/holding.rb

```diff
@@ -0,0 +1,15 @@
+module Folio
+  class Holding
+    def initialize(id:, location:)
+      @id = id
+      @location = location
+    end
+
+    def self.from_dynamic(json)
```

Comment:
this is a minor thing, but why `dynamic`? I'd just call these methods `from_json` (unless there's some convention here that I don't know about)

Reply:
That isn't json, it's a ruby hash.

Reply:
That's what I said about `#as_json`, but apparently that's an ActiveRecord thing for "the hash that you _then_ turn into JSON"...

Reply:
See: https://github.com/sul-dlss/searchworks_traject_indexer/blob/462a9b5f94b83c5ae1ef03548bf502432c6b7a67/lib/folio_record.rb#L137C1-L140

Reply:
i'd be fine with `#from_hash` too, fwiw

Reply:
That's fine, this is just copying how [quicktype](https://github.com/quicktype/quicktype/blob/2b8ab70650ec37fcd97e660dde62bf3d70470763/packages/quicktype-core/src/language/ruby/index.ts#L271) generates these sort of data classes.